libioc/Config/Jail/Globals.py

```diff
@@ -67,7 +67,7 @@
     "allow_mlock": None,
     "allow_mount": 0,
     "allow_mount_devfs": 0,
-    "allow_mount_fusefs": 0,
+    "allow_mount_fusefs": False,
```

Comment:
Interesting idea to actually turn the `allow_mount_*` properties into booleans. The sysctl value is an integer type, but there is a translation of boolean values happening. We should just be consistent and change similar properties among this one.

Reply:
OK. I think should keep a consistency for code(For example, 0 or 1 integer only used, or boolean type only used).
If possible, To refactoring for future, I think should only use boolean type.

Reply:
Anyway, I do fix such `allow_mount.*` parameters from 0,1 to True, False.